sdcm/sct_config.py

```diff
@@ -1758,6 +1758,8 @@ def __init__(self):
             self['user_prefix'] = "{}-{}".format(user_prefix, version_tag)[:35]
         else:
             self['user_prefix'] = user_prefix[:35]
+        if 'k8s' in cluster_backend:
+            self['user_prefix'] = self['user_prefix'].lower()
```

Comment:
we might do this closer to where it goes to k8s ? i.e. this is related to something new, we didn't had this before...

Reply:
It appeared with the addon of the K8S monitoring functional tests which use DB cluster name for K8S object creation.
There were no other such cases before.

Reply:
Not actual anymore. Just uploaded really correct fix.